js/src/tooltip.js

```diff
@@ -245,83 +239,85 @@ class Tooltip extends BaseComponent {
       throw new Error('Please use show on visible elements')
     }
 
-    if (this.isWithContent() && this._isEnabled) {
-      const showEvent = EventHandler.trigger(this._element, this.constructor.Event.SHOW)
-      const shadowRoot = findShadowRoot(this._element)
-      const isInTheDom = shadowRoot === null ?
-        this._element.ownerDocument.documentElement.contains(this._element) :
-        shadowRoot.contains(this._element)
+    if (!(this.isWithContent() && this._isEnabled)) {
+      return
```

Comment:
This is not covered by the tests, I wonder if we could add a new test?

Reply:
This check was already there, just the way of implementation is changed:
https://github.com/twbs/bootstrap/blob/56fdcf9e3510a53e73304869fbb444227b1451eb/js/src/tooltip.js#L248